tests/ml_tools/test_checkpointing.py

```diff
@@ -149,3 +174,26 @@ def loss_fn(model: QuantumModel, data: torch.Tensor) -> tuple[torch.Tensor, dict
     for ckpt in ckpts:
         loaded_model, optimizer, _ = load_checkpoint(tmp_path, BasicQNN, optimizer, ckpt, "")
         assert torch.allclose(loaded_model.expectation(inputs), model.expectation(inputs))
+
+
+def test_basicqQM_save_load_legacy_ckpts(BasicQuantumModel: QuantumModel, tmp_path: Path) -> None:
```

Comment:
```suggestion
def test_basic_qm_save_load_legacy_ckpts(BasicQuantumModel: QuantumModel, tmp_path: Path) -> None:
```

Reply:
Changed the name of all tests to match this style as well